lib/pavilion/plugins/commands/cat.py

```diff
@@ -54,11 +54,8 @@ def print_file(file):
 
     try:
         with file.open() as file:
-            while True:
```

Comment:
The way you did this before was better; you don't want to read a whole arbitrary file into memory at once.
Just add `end=""` to your print statement. (fprint passes extra keyword arguments to print())